sdk/core/azure-core/azure/core/configuration.py

```diff
@@ -113,8 +113,8 @@ class ConnectionConfiguration(object):
     """
 
     def __init__(self, **kwargs):
-        self.timeout = kwargs.pop('connection_timeout', 300)
-        self.read_timeout = kwargs.pop('read_timeout', 300)
-        self.verify = kwargs.pop('connection_verify', True)
-        self.cert = kwargs.pop('connection_cert', None)
-        self.data_block_size = kwargs.pop('connection_data_block_size', 4096)
+        self.timeout = kwargs.pop("connection_timeout", 300)
```

Comment:
We should make sure this gets typed. Moving things from `**kwargs` to (typed) keyword-only arguments should take care of that, though...

Reply:
Done, moved them to full kwarg-only + typing